sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -215,6 +215,16 @@ public static Read readResources() {
     return new Read();
   }
 
+  /**
+   * Search resources from a PCollection
+   *
+   * @return the search
+   * @see Search
+   */
+  public static Search searchResources(String fhirStore) {
```

Comment:
Should we pass the queries in as well, or is it part of `fhirStore`?

Reply:
I just realized that your case is totally different from what I thought, I expect a single query is used to generate the input to the pipeline, while it looks like we are doing something in reverse here, the search parameters are from the input. Can you confirm this is indeed the use case you have in mind? Please feel free to reach out to me offline.

Reply:
I'll reply in an email offline.